app/src/main/assets/html/error/js/error.js

```diff
@@ -0,0 +1,12 @@
+var mErrorURL;
+var mErrorDescription;
+function updateMessage(errorURL, errorDescription) {
+    mErrorURL = errorURL;
+    document.getElementById("errorURL");
+    if (document.getElementById("errorURL"))
+        document.getElementById("errorURL").innerHTML = errorURL;
```

Comment:
feel free to use `textContent` instead of `innerHTML` (for sanitisation/escaping purposes)